src/constants.h

```diff
@@ -2,6 +2,8 @@
 #ifndef CONSTANTS_H
 #define CONSTANTS_H
 
+#include "predicatesDLLExport.h"
```

Comment:
This include shouldn't be necessary -- we're not exporting anything from this header.